tests/python/PEP8NotebookBearTest.py

```diff
@@ -0,0 +1,104 @@
+from tests.LocalBearTestHelper import verify_local_bear
+from bears.python.PEP8NotebookBear import PEP8NotebookBear
+
+
+def prepare_string(file):
+    """
+    Utility function to prepare Notebook strings for further processing.
+
+    replace() necessary to avoid
+    nbformat.reader.NotJSONError: Notebook does not appear to be JSON
+    which is caused by
+    json.decoder.JSONDecodeError: Invalid control character
+    which is caused by the \n characters inside the input cells
+    """
+    return file.replace("\n\"", "\\n\"")
```

Comment:
Omg, I wrote a function for that, yes. At the time of writing I did not see through it...
Thanks! :+1: